src/main/java/seedu/address/logic/commands/SearchPublicAddressCommand.java

```diff
@@ -13,12 +14,12 @@ public class SearchPublicAddressCommand extends Command {
     public static final String COMMAND_WORD = "searchPublicAddress";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-        + ": Searches for a public address and returns the user, network and tag "
+            + ": Searches for a public address and returns the user, network and tag "
             + "throughout all the public addresses of all networks in the address book.\n"
-            + "Parameters: PUBLIC_ADDRESS (must be a string) "
-            + PREFIX_PUBLIC_ADDRESS + "[PUBLIC_ADDRESS]\n"
-            + "Example: " + COMMAND_WORD + " " + PREFIX_PUBLIC_ADDRESS
-            + "0x28f91d6e72eaf4372892e6c6e45dc41b574163e9fcdf94f4997958b46d772fa2";
+            + "Parameters: PUBLIC_ADDRESS (must be a string) " + PREFIX_PUBLIC_ADDRESS
+            + "[PUBLIC_ADDRESS]\n"
```

Comment:
Square brackets are reserved for optional parameters, should instead be PREFIX + PUBLIC ADDRESS.

Reply:
Thx sir, very detailed, very precise.